config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
   mount Ckeditor::Engine => "/ckeditor"
   scope "(:locale)", locale: /en|vi/ do
-    root "home#index"
-    devise_for :users, controllers: {sessions: "users/sessions"}
-    namespace :admin do
+    as :user do
+      devise_for :users, path: "", path_names: {sign_in: "login",
```

Comment:
Để devse_for theo mặc định ở ngoài đi không là failure login đó. Đã merge chạy rồi thì chú chỉ custom chứ đừng thay đổi hoàn toàn vậy.